File: android/app/src/main/java/com/vihaangiri/jamai/JamAiApp.java
package com.vihaangiri.jamai;

import android.app.Application;
import android.webkit.WebView;

public class JamAiApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        WebView.setWebContentsDebuggingEnabled(true);
    }
}

File: android/app/src/main/kotlin/com/vihaangiri/jamai/JamAiApp.kt
package com.vihaangiri.jamai

import android.app.Application
import android.webkit.WebView

/**
 * JAM AI - Application Entry Point in Kotlin
 * Architect: Vihaan Giri
 */
class JamAiApp : Application() {

    override fun onCreate() {
        super.onCreate()
        // Enable WebView debugging in debug builds
        if (BuildConfig.DEBUG) {
            WebView.setWebContentsDebuggingEnabled(true)
        }
    }
}
